ral-facilities/datagateway
# Adding the Cart to search #1092

## Description
Adding the cart to search.

- Adding the cart to the end of the search app bar  
- Moving aria-labels to the default.json for cart aria-labels 

## Testing instructions
Add a set up instructions describing how the reviewer should test the code
Check the cart colour in all the different themes 


## Agile board tracking
closes #1092
